cumulus_library/statistics/psm.py

```diff
@@ -114,16 +120,9 @@ def _get_sampled_ids(
         :param is_positive: defines the value to be used for your filtering column
         """
         df = cursor.execute(query).as_pandas()
+        df = df.sort_values(by=[self.config.primary_ref])
         df = (
-            df.sort_values(by=[self.config.primary_ref])
-            # .reset_index()
-            # .drop("index", axis=1)
-        )
-
-        df = (
-            # TODO: flip polarity of replace kwarg after increasing the size of the
-            # unit testing data
-            df.sample(n=sample_size, random_state=self.config.seed, replace=True)
+            df.sample(n=sample_size, random_state=self.config.seed, replace=False)
```

Comment:
i hear you - i kind of like the explicit, if only because it was specified that the PSM sampling would not be valid with replace, but that could also be a comment at the top of the class.